Traceback (most recent call last):
  File "c:\Users\Carl\.vscode\extensions\ms-python.python-2022.20.1\pythonFiles\shell_exec.py", line 26, in <module>
    subprocess.check_call(shell_args, stdout=sys.stdout, stderr=sys.stderr)
  File "C:\Python310\lib\subprocess.py", line 369, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['C:/Python310/python.exe', '-m', 'pip', 'install', '-U', 'tensorboard', 'C:/Windows/TEMP/tmp-24288QV6JuJDUvDDq.log']' returned non-zero exit status 1.
